### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by tmampa**
In this milestone we worked on the 
-Magazine section
-Newspaper section 
using 
-Bootstrap 
-Css  
-Html.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -2,22 +2,141 @@
...
+  width: 100%;
+}
+
+.subimage2 {
+  top: 30px;
+  left: 80px;
+}
+
+.subdiv2 {
+  width: 100%;
```
Comment (Berabjesus) [inline css/style.css]: - [ ] You have a lot of `width: 100%;` rules in your css. Use Bootstrap's `w-100`

File: css/style.css [hunk 2]
```diff
@@ -2,22 +2,141 @@
...
+  right: 45%;
+  bottom: -16px;
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
+  text-align: center;
+  font-weight: bold;
```
Comment (Berabjesus) [inline css/style.css]: - [ ] use bootstrap classes for this and other similar rules when appropriate.

File: index.html [hunk 3]
```diff
@@ -60,66 +72,1830 @@
...
+                </section>
+                <div class="gray w-100 d-md-none"> <br></div>
+            </div>
+        </div>
+
+
+
+        <!-- In the magazine -->
+        <div class="n-title flex-xs jc-c ai-c d-flex justify-content-center mt-3 mb-3 margin-bottom fs-3 text "><span
+                class="icon-n"></span>IN THE MAGAZINE</div>
```
Comment (Berabjesus) [inline index.html]: - [ ] ## Your "in the magazine" section gets distorted on medium screen size. Notice how the section fills the width. I took the screenshots on 856 px.
### original 👇 
image

### yours 👇 
image

- [ ] ## On mobile screen size the pictures should fill the entire width (with a small horizontalal padding )
### Original 👇 
image

### yours 👇 
image

File: index.html [hunk 4]
```diff
@@ -60,66 +72,1830 @@
...
+                <div>
+                    <span class="red fs-6 text text-light"> COVER </span>
+                    <span> <a href="#" class="bg-light text-danger fs-5 text"> NEWS</a></span>
+                    <h4><a href="#">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></h4>
+                </div>
+            </div>
+        </div>
+        <!-- editors pick -->
+        <div class="n-title flex-xs jc-c ai-c d-flex justify-content-center mt-3  mb-3 margin-bottom fs-3 text"><span
+                class="icon-n "></span>EDITOR'S PICK</div>
```
Comment (Berabjesus) [inline index.html]: - [ ] images in your "editors pick" section is not correctly styled on medium screen size. Notice how the images are bigger. I took the screenshots on 856 px.
### original 👇 
image

### yours 👇 
image

File: index.html [hunk 5]
```diff
@@ -60,66 +72,1830 @@
...
+                    <div class="text-box d-flex flex-column align-items-center align-items-md-start p-4">
+                        <strong class="fs-4">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</strong>
+                        <p>All Access + Weekly Delivery.
+                            Choose A Membership That's Perfect for You!</p>
+                        <button class="btn bg-news text-white fw-bold">SUBSCRIBE &gt;</button>
+                    </div>
+                </div>  
+            </div>                        
+        </section>
+        <div class="gray w-100 d-md-none"> <br></div>
```
Comment (Berabjesus) [inline index.html]: - [ ] subscribe section should have a clickable button. 
- [ ] The text is hidden. 
### original 👇 
image

### Yours 👇 
image

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: CHANGES REQUIRED :warning: :construction: 
**Hi @tmampa  and @danmainah very nice project you have done so far. Good job with the professional PR :tada:** 

### You do, however, have a few changes to carry out before we proceed so please take a look at the things you have to work on and their respective review comments below.

- [ ] Make sure you have a descriptive README in place and that you follow the [repository good practices.](https://microverse.zendesk.com/hc/en-us/articles/360057489093-General-Requirements-for-Microverse-Projects). currently, your readme does not have a detailed section on how to get started and clone this project. 

- [ ] Whenever necessary, use Bootstrap CSS classes everywhere for styling. You have used `display: flex;` and `justify-content: flex-end;` Use bootstrap classes for these, like `d-sm-flex` (check comments below for similar details)

- [ ] In comparison with the [original page](https://web.archive.org/web/20210120125445/https://www.newsweek.com/), the "In the magazine's" section gets distorted on medium screen size. (check comment below for detail)
- [ ] In comparison with the [original page](https://web.archive.org/web/20210120125445/https://www.newsweek.com/), the "Editors pick" section has a wrong styling of pictures on medium and mobile mode. (check comment below for detail)
- [ ] The subscribe section should have a button and should be styled properly.(check comment below for detail)

### ** ❗ You have two (2) remaining reviews. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @tmampa and @danmainah :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request closed without merging**